packages/alfa-rules/src/sia-r83/rule.ts

```diff
@@ -60,11 +61,13 @@ export default Rule.Atomic.of<Page, Text>({
       },
 
       expectations(target) {
+        const parent = target.parent();
+
         return {
           1: expectation(
-            wrapsText(device)(target),
-            () => Outcomes.WrapsText,
-            () => Outcomes.ClipsText
+            parent.every(and(isElement, not(wrapsText(device)))),
```

Comment:
```suggestion
            target.parent({ flattened: true, nested: true }).every(and(isElement, not(wrapsText(device)))),
```
Since `parent` is only used once, no need to create a variable for it. Plus, we should go up with the same options (flattened, nested) that we went down in the applicability.

Reply:
And I'm actually thinking we messed up the quantifier here 😕 
If there is no parent (which is not really possible due to the applicability, but whatever), the rule should pass (nothing clips). But `parent.every` is `true` on `None` which ends up returning the error…
So it should be `parent.some` instead (I think we messed up when we moved the negations around and forgot to negate the quantifier).